admin/client/src/styles/legacy/_forms.scss

```diff
@@ -409,7 +409,7 @@ form.small .field, .field.small {
 				border: none;
 			}
 			&.loading {
-				background: transparent url(../images/network-save.gif) no-repeat $grid-x center;
+				background: transparent url(../images/network-save.gif) no-repeat right $grid-x center;
```

Comment:
what does the change from `<num> center` to `right <num> center` mean for positioning? Is one of those three variables now needing to be removed?